stairctl/src/main/java/bio/terra/stairctl/commands/ConnectCommands.java

```diff
@@ -0,0 +1,65 @@
+package bio.terra.stairctl.commands;
+
+import bio.terra.stairctl.ConnectParams;
+import bio.terra.stairctl.StairwayService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.shell.standard.ShellComponent;
+import org.springframework.shell.standard.ShellMethod;
+import org.springframework.shell.standard.ShellOption;
+
```

Comment:
Spring Shell is pretty cool, I didn't know this existed...  So the only state we track during the session currently connection state, but if we needed to track more session state would it go in `StairwayService`?

Reply:
Probably, although we could add other `@Components` if we had other kinds of state.